### ISSUE TITLE
Handling of login token: need method to refresh token

### ISSUE THREAD

**Issue by gruijter** [simplyGits/MagisterJS#100]
Since version alpha4 it is possible to login using a token. This token however expires after 1 or 2 hours. Is it possible to provide a method that will refresh the token in time (preferably fully automatic :) )?

**Comment by Sj3rd** [comment]
You can refresh the token by just logging in again with a username and password :)

**Comment by gruijter** [comment]
Sure, but then I get #91

**Comment by Sj3rd** [comment]
Well, the token refresh feature will be doing that. I cant find any code in the magister web client which refreshes the token...

I think its a better idea to try to fix your error.

**Comment by lieuwex** [comment]
@Sj3rd Indeed, the way I would implement this feature would be by relogging in. (which kinda is already slightly supported, or at least it was on v1, I don't remember if I ported it over to v2).

**Comment by Argetan** [comment]
So there is, as far as you are aware, no way to make a token last longer? Say a couple days or a week? This could allow people to authenticate once and then reuse the token for subsequent logins. However, if the token needs to be "refreshed" every couple hours it would just be a nuisance. 

I also came across [this page](https://www.magister.nl/overstappen-op-twee-factor-authenticatie-2fa-voor-magister-desktop/) a couple days ago, will there be any complications for `Magister.js`?
